### PR TITLE
Creating 1.4.1 build (based on 1.4.0.2); bump default build to 1.4.2

### PR DESCRIPTION
**Pull request by ceejatec**

### CODE REVIEW

File: manifest/1.4.1.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  <project name="otto" path="godeps/src/github.com/robertkrimen/otto" remote="couchbasedeps" revision="5282a5a45ba989692b3ae22f730fa6b9dd67662f" />
+  <project name="pkg" path="godeps/src/github.com/coreos/pkg" remote="coreos" revision="160ae6282d8c48a33d8c150e4e4817fdef8a5cde" />
+  <project name="retriever" path="godeps/src/github.com/couchbase/retriever" revision="19c5a5d92a2f34fb96ae91d26901e4a7076b8020" />
+  <project name="service" path="godeps/src/github.com/kardianos/service" remote="kardianos" revision="2954cfdd7b0c8ab45ef2aa22a44b5f086201836f" />
+  <project name="sg-bucket" path="godeps/src/github.com/couchbase/sg-bucket" revision="a22e64010d08574baa33332695abb27c8e50d251" />
+  <project name="sg-replicate" path="godeps/src/github.com/couchbaselabs/sg-replicate" remote="couchbaselabs" revision="24b8252856d71a541cbfeb2203cf7c0ce3634b2b" />
+  <project groups="notdefault,sg-accel" name="sync-gateway-accel" path="godeps/src/github.com/couchbaselabs/sync-gateway-accel" remote="couchbaselabs_private" revision="82be89009d2d9c0c3d546dd0f7d19428376497e7" upstream="master" />
```
Comment (adamcfraser) [inline manifest/1.4.1.xml]: Looks like the accel commit in the 1.4.1 manifest still needs to get rolled back to 7fe4854f1e98ea2ec0ab548bd763030cc133b61b.

Comment (ceejatec) [reply manifest/1.4.1.xml]: I pushed a second commit already to fix that; is it not showing up?

Comment (adamcfraser) [reply manifest/1.4.1.xml]: I see the commit that fixes it in manifest/1.4.0.xml, but I don't see it as fixed in manifest/1.4.1.xml

Comment (ceejatec) [reply manifest/1.4.1.xml]: Ah, crap, I made the wrong change. I'll fix.

**Pull request merged**